### DISCUSSION TITLE
Right click support with a touchscreen

### DISCUSSION THREAD

**Discussion by ld892012** [BuddiesOfBudgie/budgie-desktop#504] [Q&A]
How i can do a right click with a touchscreen in Budgie ? A long press doesn't do anything
I tried with Solus and Fedora but with same effect, but Gnome flavor of these OS works
This kind of problems is due to desktop environment or something else ?

**Comment by EbonJaeger** [comment]
It's been a long time since I used my laptop with a touchscreen, but I *think* a two-finger tap works as a right-click. I might be confusing that with a trackpad, though.

**Reply by ld892012** [reply]
This is right for a trackpad but not for the touchscreen, it's strange because scrolling works but not right click
